Error in rule create_database:
    jobid: 0
    output: output/db_name.txt

RuleException:
ModuleNotFoundError in line 15 of /home/ddhoogduin/programs/KeyGenes-dataprocessor-v0.2/Snakefile:
No module named 'pymysql'
  File "/home/ddhoogduin/programs/KeyGenes-dataprocessor-v0.2/Snakefile", line 15, in __rule_create_database
  File "/home/ddhoogduin/programs/KeyGenes-dataprocessor-v0.2/Database.py", line 48, in add_db
  File "/home/ddhoogduin/programs/KeyGenes-dataprocessor-v0.2/Database.py", line 20, in connect
  File "/home/ddhoogduin/.local/lib/python3.6/site-packages/sqlalchemy/engine/__init__.py", line 479, in create_engine
  File "/home/ddhoogduin/.local/lib/python3.6/site-packages/sqlalchemy/engine/strategies.py", line 87, in create
  File "/home/ddhoogduin/.local/lib/python3.6/site-packages/sqlalchemy/dialects/mysql/pymysql.py", line 62, in dbapi
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Exiting because a job execution failed. Look above for error message
